src/fhost_fpga.py

```diff
@@ -150,36 +157,51 @@ def __init__(self, host, katcp_port=7147, boffile=None, connect=True, config=Non
             self.min_load_time = None
             self.network_latency_adjust = None
 
+        self._pfb_of_counts = {}
+        self._ct_counts = {}
+        self._qdr_counts = {}
+        self.fnums = int(configd['fengine']['f_per_fpga'])
```

Comment:
This is already stored in self.num_fengines? Also, I thing the config passed in is already at the level of ['fengine'], so that part could have been dropped.